services/web/client/source/class/osparc/desktop/preferences/pages/ProfilePage.js

```diff
@@ -87,6 +87,20 @@ qx.Class.define("osparc.desktop.preferences.pages.ProfilePage", {
 
       box.add(new qx.ui.form.renderer.Single(form));
 
+      const expirationLayout = new qx.ui.container.Composite(new qx.ui.layout.HBox(5)).set({
+        paddingLeft: 16,
+        visibility: "excluded"
+      });
+      expirationLayout.add(new qx.ui.basic.Label(this.tr("Expiration date:")));
+      const expirationDate = new qx.ui.basic.Label();
+      expirationLayout.add(expirationDate);
+      const infoLabel = this.tr("Please, contact us at email:<br>");
```

Comment:
Maybe "Please, contact us *by* email" sounds better...

Reply:
Google says 20M vs 1M, you win